src/agent/driver-test-runner.js

```diff
@@ -127,7 +137,21 @@ export class DriverTestRunner {
           },
         });
       }
-    } else if (!capabilities.atName) {
+    } else if (!atName) {
+      return;
+    }
+    throw new Error(`Unable to ensure proper settings. Unknown atName ${capabilities.atName}`);
```

Comment:
This error would throw, I think, since capabilities isn't defined anymore
```suggestion
    throw new Error(`Unable to ensure proper settings. Unknown atName ${atName}`);
```